.github/workflows/haiku.yml

```diff
@@ -0,0 +1,53 @@
+name: Build (Haiku @QEMU)
+
+on:
+  # push:  # Disabled ci on push because of long build times
+  pull_request:
+    paths:
+      - '**/haiku/*'
+      - '.github/workflows/haiku.yaml'
```

Comment:
Looks like there is some inconsistency with `.yml` vs `.yaml` in this folder -- I used `.yaml` for my example because I was talking about `ps2.yaml`, but this file here is `haiku.yml` without the `a`

(I guess it'd probably be worth making all the filenames consistent to avoid confusion like this?)

Reply:
Looks like only the vita rename got committed, but ps2 and psp are still .yaml ^^;

Reply:
I need new glasses. Thanks :)